apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx

```diff
@@ -72,7 +72,7 @@ export const ReceiptsTable: React.FC<IContextProps> = ({ context }) => {
 
     const [modalIsVisible, setModalIsVisible] = useState(false)
     const [detailedReceipt, setDetailedReceipt] = useState<BillingReceiptType>(null)
-    const [period, setPeriod] = useState(dayjs(get(context, ['lastReport', 'period'], null)))
+    const [period, setPeriod] = useState(dayjs(get(context, ['lastReport', 'period'], undefined)))
```

Comment:
OK, but `undefined` is the default value for 3rd parameter, so this is redundancy :)
https://lodash.com/docs/4.17.15#get